geospaas_processing/cli/copy.py

```diff
@@ -0,0 +1,108 @@
+"""
+Download files that are selected from the database using input criteria.
+"""
+import argparse
+import json
+import ntpath
+import os
+from os import path
```

Comment:
No, I have intentionally used it this way in order not to get confused in the tests for the definition of `path.is_file` which is used in the assertion of the tests. This way is much more clear with the correct mocking of usage (instead of definition) of path.

Reply:
That's not a valid justification. It's not a problem to mock `os.path`.
I won't argue about that again, please read https://docs.python.org/3/library/unittest.mock.html#where-to-patch if you have doubts.

Reply:
Discussed. Done in the next commit.